Honor FHS to please debian

--- a/cap32.cfg
+++ b/cap32.cfg
@@ -51,7 +51,7 @@
 joystick_vkeyboard_button=10
 # resources_path
 #   path to resources (menu images...). Default to $APP_PATH/resources
-resources_path=
+resources_path=/usr/share/caprice32/resources
 # boot_time
 #   Estimated time in video frames the CPC takes to boot.
 #   Caprice will emulate this number of frames before starting to send a provided autocmd.
@@ -200,7 +200,7 @@
 #   Directory where ROMs are stored. Default to $APP_PATH/rom
 #   Special keyword DEFAULT will be replaced by amsdos.rom except if model is 464.
 #   That's because 464 without disks had no AMSDOS (cf. https://github.com/ColinPitrat/caprice32/issues/227)
-rom_path=
+rom_path=/usr/share/caprice32/rom
 slot00=
 slot01=
 slot02=
--- a/cap32.cfg.tmpl
+++ b/cap32.cfg.tmpl
@@ -51,7 +51,7 @@
 joystick_vkeyboard_button=10
 # resources_path
 #   path to resources (menu images...). Default to $APP_PATH/resources
-resources_path=__SHARE_PATH__/resources
+resources_path=/usr/share/caprice32/resources
 # boot_time
 #   Estimated time in video frames the CPC takes to boot.
 #   Caprice will emulate this number of frames before starting to send a provided autocmd.
@@ -172,16 +172,16 @@
 max_track_size=5990
 # cart_path
 #   Path to cartridges (Plus range). Default to $APP_PATH/cart
-cart_path=__SHARE_PATH__/cart
+cart_path=
 # snap_path
 #   Path to snapshots. Default to $APP_PATH/snap
-snap_path=__SHARE_PATH__/snap
+snap_path=
 # dsk_path
 #   Path to disks. Default to $APP_PATH/disk
-dsk_path=__SHARE_PATH__/disk
+dsk_path=
 # tape_path
 #   Path to tapes. Default to $APP_PATH/tape
-tape_path=__SHARE_PATH__/tape
+tape_path=
 fmt02=
 fmt03=
 fmt04=
@@ -193,14 +193,14 @@
 printer_file=./printer.dat
 # sdump_dir
 #   Directory where screenshots will be found. Default to $APP_PATH/screenshots
-sdump_dir=__SHARE_PATH__/screenshots
+sdump_dir=
 
 [rom]
 # rom_path
 #   Directory where ROMs are stored. Default to $APP_PATH/rom
 #   Special keyword DEFAULT will be replaced by amsdos.rom except if model is 464.
 #   That's because 464 without disks had no AMSDOS (cf. https://github.com/ColinPitrat/caprice32/issues/227)
-rom_path=__SHARE_PATH__/rom
+rom_path=/usr/share/caprice32/rom
 slot00=
 slot01=
 slot02=
